static/src/stylesheets/module/identity/_switches.scss

```diff
@@ -226,3 +226,12 @@ $checkbox-size: $gs-gutter / 1.25;
         margin-right: .25em;
     }
 }
+
+@include mq(desktop) {
```

Comment:
this shouldn't be part of `switches.css` but rather live in `manage-account-tab` probably?

Reply:
Good call, I'm going to put it in `_identity.scss` as I think `manage-account-tab` was intended just for the account tab.